samples/CommunityToolkit.Maui.Sample/Pages/Views/SnackBarPage.xaml.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.Text;
+using System.Threading.Tasks;
+using CommunityToolkit.Maui.Controls.Snackbar;
+using CommunityToolkit.Maui.Extensions;
+using Microsoft.Maui;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Graphics;
+
+namespace CommunityToolkit.Maui.Sample.Pages.Views;
+
+public partial class SnackBarPage : BasePage
+{
+	private Snackbar? snackbarWithAnchor;
+	public SnackBarPage()
+	{
+		InitializeComponent();
+		this.Anchor1 ??= new Microsoft.Maui.Controls.Button();
```

Comment:
I don't like that we have to add it for each control, because we know that after InitializeComponent() the control cannot be null. are there any hacks to avoid it?

Reply:
It's the only option we have right now. 

The MAUI team is aware of it: https://github.com/dotnet/maui/issues/1796#issuecomment-946348892

Reply:
You can just use `= new()` to save some keystrokes